### ISSUE TITLE
NullReferenceException in XwaHookDialog ctor

### ISSUE THREAD

**Issue by JeremyAnsel** [MikeG621/YOGEME#27]
Reported by Darksaber at the XWAUP forum: [YOGEME by JaggedFel](https://www.xwaupgrade.com/phpBB3/viewtopic.php?f=9&t=12313)

In [XwaHookDialog.cs](https://github.com/MikeG621/YOGEME/blob/master/XwaHookDialog.cs), there is this code:
```csharp
_fileName = mission.MissionPath.Replace(".tie", ".ini");
...
StreamReader srMission = null;
if (File.Exists(_fileName)) srMission = new StreamReader(_fileName);
…
if (srMission != null)
{
    #region read
    …
    #endregion
}

srMission.Close();
```
When the ini file doesn't exist, `srMission` is null and a NRE is thrown at `srMission.Close()`.

To fix the issue, place `srMission.Close();` at the end of the `if (srMission != null)` block.

**Comment by MikeG621** [comment]
Bah.  Fixing, will be up in a couple minutes.
--
Michael Gaisser

**Comment by MikeG621** [comment]
Fixed in 2d7678c and released
